IM_calculation/Advanced_IM/Models/Spear_3D_Model/run.py

```diff
@@ -0,0 +1,311 @@
+import argparse
+import glob
+import os
+import pandas as pd
+import subprocess
+
+import numpy as np
+
+

```

Comment:
the issue is that 3D models perform differently, compare to 2D.
and there is only one 3D models at the moment so i wasnt sure if i should have a lib.
But if you prefer, i can make a runlibs_3D for it.

Reply:
Ahhh I see. Makes sense - we can create a 3D lib when we add the next 3D model. I think it can at least use the argparse from it. It may make sense to have 2D and 3D content in run_libs and have 2D/3D specific functions.

Reply:
I'm generally in favour of separate 2d, 3d and common libs.
But if we only have one 3d model atm, then having 3d funcs with the model is fine, they can be moved out later as required